### DISCUSSION TITLE
Accessing a Gate in the Family Crypt around Lochport Cemetery: Is It Possible?

### DISCUSSION THREAD

**Discussion by Vecna-max** [flareteam/flare-game#971] [Q&A]
I've repeatedly searched for a way to open a family crypt gate in Lochport Cemetery but can't find anything. Is the gate just decorative, or am I missing something? Is there a way to get in, or is it supposed to be inaccessible? The crypt is at the southernmost end, near an above-ground coffin. Does anyone know if it's possible to access that area? 
To clarify, this gate is directly opposite Ardwyn's Coffin and not the one that can be opened with the lever. If I can, I’ll try to add an image for reference.

![Family Crypt](https://github.com/user-attachments/assets/3a317fb6-ff2a-4b9e-94d7-9b75a97e8921)

**Comment by dorkster** [comment]
Sorry for the confusion. That door is only decorative.

**Reply by Vecna-max** [reply]
Thank you! I really appreciate that you're answered my questions. I do have a few more questions though about the game.

First, is there a reason why there seems to be so little lore ? In the hour and 30 minutes I've played so far, I’ve only completed about three quests, and I haven’t encountered many NPCs. I understand this might be due to the game’s context and story, but I was expecting lots of lore to make up the lack of NPCs.

For example, when I explored the family crypt, it mentioned that the crypt is occupied by the man who founded Lochport, but there was no explanation about what happened to the town or him. When I visited Lochport, it was overrun with skeletons and undead, but the town itself looked untouched. The buildings were intact, nothing appeared decayed, and there were even plants growing in the garden. I would have expected some backstory or lore explaining why the town failed or how everyone became undead, but there was nothing. I would understand that there would be a significant lack of lore if the town was destroyed, but even in a destroyed town you can get clues to what happened, for example if there was a monster attack the buildings would be smashed, if they all just vanished the houses would be moldy, dusty or falling apart.

Currently, I’m in the crypt of a saint in town. On the first level, there are books everywhere, but none of them contain any lore about who the saint was or what happened there. But who knows I might know all the answers while continue playing. It just feels a bit empty in terms of world-building.

That being said, I’m still having a lot of fun with the game! I’m just curious if more lore is planned.

**Reply by dorkster** [reply]
Lore and world building simply weren't given much consideration when building the Empyrean Campaign. And there are no plans to do so (though, I'm not against it).

**Reply by Vecna-max** [reply]
I just have a lot of questions about the game, especially regarding the lore, and you stated that the lore and worldbuilding weren't given much consideration when building the Empyrean campaign and that there are no plans to develop them further, then what’s the point of the game? Is it just about killing enemies in different areas?

For instance, there are tombs like St. Maria's Tomb, family crypt, and the ruins of Lochport like what happened and who are these people especially a saint, there are also named enemies in St. Maria's Tomb that are more powerful than others, and there's no information given about them. It feels like random characters just appear out of nowhere, wielding greater powers without any context. And what about Seagate? Why is he significant? Why did he randomly appear in St. Maria tomb, why would there be so many lore-like details if they weren't given much thought during development?

**Reply by dorkster** [reply]
>then what’s the point of the game? Is it just about killing enemies in different areas?
>It feels like random characters just appear out of nowhere, wielding greater powers without any context.

It's safe to say that the game was designed by building the world around the gameplay / mechanics, as opposed to designing the world and then inserting things to fit it. So you could say the point *is* fighting random but increasingly strong enemies. My view is that this is acceptable for an action RPG, where many players will just gloss over the story anyway.

>why would there be so many lore-like details if they weren't given much thought during development?

I felt that there needed to be at least *some* flavor in the world, but at the same time, not everything needed a full explanation. And usually, the details are kept to a single area versus being something that's linked across the game world. I'd rather just leave interpretations to the player.
